lib/libraries/default/message.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  render: function(text){
+    var outer = document.createElement('div'),
+        inner = document.createElement('div'),
+        msg = document.createElement('span'),
+        height = this.height() || 140;
+
+    outer.className = this.theme();
+    inner.className = this.theme() + '-message';
+    inner.style.height = String(height + 'px');
+    inner.style.paddingTop = (height / 2 - 12) + 'px';
+    inner.style.width = String(this.width() + 'px');
```

Comment:
Concatenating with `'px'` will force the return value to be a `string` here, no need for `String()` typecasting.